drake/systems/plants/collision/ResultCollector.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+#include "drake/systems/plants/collision/DrakeCollision.h"
+#include "ResultCollector.h"
```

Comment:
I wasn't intending this to turn into fixing existing problems in BulletResultCollector, but...

(I'm going to refrain from commenting on what I think those include guidelines...)